vercel.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version": 2,
+  "name": "builder",
+  "build": {
+    "env": {
+      "NODE_PATH": "src",
+      "NODE_ENV": "development",
+      "GEN_STATIC_LOCAL": "true",
+      "PUBLIC_URL": "/",
+      "REACT_APP_DCL_DEFAULT_ENV": "dev"
```

Comment:
This is only defined in Vercel and in the examples, what is it for?

Reply:
It overrides the default environment used. The behavioir is:

- Look for an `env` query param and use that as the env
- If no param present, check the TLD (if .io/.zone -> DEV, .today/.net -> STG, .org -> PROD)
- If no TLD found, use default (which is by default PROD, but with this env we override it to DEV).

This is so vercel deployments use by default the dev env (we can always change it via query param).

The same applies for `localhost`, if no query param it will default to PROD, but with that env var we can make it default to DEV.